api/hrlh.rb

```diff
@@ -10,6 +11,14 @@ class API < Grape::API
       def database
         @database ||= Orchestrate::Application.new(ENV['ORCH_API_KEY'],ENV['ORCH_REGION'])
       end
+
+      def interview_db
+        database['interviewer']
+      end
+
+      def to_interviewer(result)
+        result.value.merge({"key"=>result.key})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.